### DISCUSSION TITLE
[hellfire] damage taken from Succubuses

### DISCUSSION THREAD

**Discussion by ggrelka** [diasurgical/DevilutionX#6684] [Q&A] [answered]
I played devilutionX 1.41 and found damage from Sucubuses  is much lower than GOG version. I saved much more potions than before.
GOG hellfire has a lot of bugs and this is maybe the result of bug fix.  I  searched changelog and jarulf guide but couldn't find something about Succubus damage or player magic resistance.  So anyone know well about that?

**Comment by AJenbo** [comment]
why not 1.5.1?

**Reply by ggrelka** [reply]
Gog save file was not compatible with 1.5.1 version and there was some item name problem..  For example) Godly Plate (original) -> Godly Full Plate Mail (1.5.0)  / Kings Axe of Haste (original) -> Kings Great Axe of Haste (1.5.0)  Is it intentional?

**Reply by AJenbo** [reply]
Please share the save and we will investigate the issue

**Reply by ggrelka** [reply]
Sorry.. it was my mistake that I missed one mpq file. well compatible with 1.5.1 version.

**Comment by ggrelka** [comment]
The difference between GOG and devilutionX seems to be monster's magic To Hit.  In GOG Soul burner hit almost every shot perfect, whereas missed quite a few shot in devilutionsX. Maybe there seems to be a bug about To Hit calculation.

**Reply by StephenCWills** [reply, marked as answer]
We did fix a bug where slow missiles like Blood Star had inflated CTH because the game would execute multiple hit checks in the same tile. See #1531.

**Reply by ggrelka** [reply]
Thanks for answering! That's what I wanted to know.
